static/js/stream_muting.js

```diff
@@ -67,4 +68,24 @@ export function update_is_muted(sub, value) {
         )}'] #sub_is_muted_setting .sub_setting_control`,
     );
     is_muted_checkbox.prop("checked", value);
+
+    $(".mute-note").toggleClass("hide-mute-note", !value);
+    $(".sub_notification_setting").toggleClass("muted-sub", value);
+    $(".sub_notification_setting").find("input[type='checkbox']").prop("disabled", value);
```

Comment:
These changes to live update the UI in stream settings overlay would be better to have in a separate commit.